apps/nextra/pages/en/build/smart-contracts/book/structs-and-resources.mdx

```diff
@@ -421,9 +424,96 @@ module 0x2::m {
 }
 ```
 
+## Positional Structs
+
+_Since language version 2.0_
+
+A struct can be declared to have _positional fields_, fields which are not named
+but numbered. Positional structs behave similar to regular structs,
+except providing a different syntax which is more adequate for use cases with 
+only a few fields.
+
+Fields of positional structs are assigned in the order they appear. In the below
+example, field `0` is of type `u64` and field `1` of type `u8`:
+
+```move
+module 0x2::m {
+  struct Pair(u64, u8);
+}
+```
+
+Abilities for positional structs are declared _after_ and not before the field list,
+which is expected to be short:
+
+```move
+module 0x2::m {
+  struct Pair(u64, u8) has copy, drop;
+}
+```
+
+For pure type tags, often used for phantom types in Move code, the list of arguments
+can be also completely omitted:
+
+```move
+module 0x2::m {
+  struct TypeTag has copy, drop;
+}
+```
+
+Values of positional structs are created and deconstructed similar as function calls,
```

Comment:
```suggestion
Values of positional structs are created and deconstructed as shown below:
```

Reply:
(deconstructed similar as function calls is a bit weird, because we do not have `function_call(arguments) = value` syntax).